blockchain/standalone/subsidy.go

```diff
@@ -317,7 +317,7 @@ func (c *SubsidyCache) CalcStakeVoteSubsidy(height int64) int64 {
 // the height at which voting begins will return zero.
 //
 // This function is safe for concurrent access.
-func (c *SubsidyCache) CalcTreasurySubsidy(height int64, voters uint16) int64 {
+func (c *SubsidyCache) CalcTreasurySubsidy(height int64, voters uint16, isTreasuryEnabled bool) int64 {
```

Comment:
Can you also comment on the significance of isTreasuryEnabled?

Reply:
Added comment.